app/components/cases/new-subcase.js

```diff
@@ -158,6 +158,8 @@ export default class CasesNewSubcase extends Component {
     } else {
       subcase.linkedPieces = pieces;
     }
+    // TODO KAS-3484 is there a situation where we don't have to inherit the governmentAreas?
+    subcase.governmentAreas = await latestSubcase.governmentAreas;
```

Comment:
This needs to support an edge-case where an OVRB user creates a case (niet via MR) and adds governmentAreas.

If a kanselarij user then adds a subcase later, I would copy the existing governmentAreas from the case. Since `latestSubcase` will be undefined in this scenario, we'll need an extra step here

Reply:
And actually, if a case is created by OVRB, the governmentAreas remain editable from the publication view even after a subcase has been added. So we need to decide between:

1. only keeping this change on the case level and not propagating (i.e., the current situation)
2. propagating this change on the case level back to all subcases
3. propagating this change on the case level back to the latest subcase 
4, Simply not allowing editing of the governmentAreas on the case level if there is at least 1 subcase

In my opinion, 1 is dangerous in terms of data consistency, also w.r.t. disclosure via OSLO. 2 and 3 could work if the users want it this way. 4 seems the easiest and safest option to me.

Reply:
Not following completely.
Kanselarij shouldn't add subcases on a case created by ovrb.
It's possible, but they shouldn't. That's not the proper workflow.
The correct way would be to have a case/subcase created by kanselarij, and connect that to the existing publication via the feature from documents on the agendaitem. (ovrb case will be deleted then)

The flag to determine if ovrb can edit the government areas is the existence of agendaitem.
So if in your example the subcase would be on the agenda, it would no longer be editable.

Reply:
> Kanselarij shouldn't add subcases on a case created by ovrb.
> It's possible, but they shouldn't. That's not the proper workflow.

Ok, so this scenario shouldn't happen in practice. However, I feel weird leaving this in a way so the data could end up in an inconsistent state, even though it's not supposed to happen.

I do recall someone from OVRB saying in a meeting they sometimes start preparing the "publication" `case` for certain agenda items beforehand if they know it's going to be a lot of work. So it might not be so uncommon after all...

Also, the fix seems quite simple to avoid the issue altogether: 

1. add the something along the lines of 

```
else if (this.args.case.governmentAreas) {
   subcase.governmentAreas = await this.args.case.governmentAreas;
}
```
here: https://github.com/kanselarij-vlaanderen/frontend-kaleidos/blob/5a61282030b33e5088d0188e843048ac51a4da4b/app/components/cases/new-subcase.js#L130

2. include the existence of a subcase in the editable flag for OVRB.

Since it's minimal effort to include it, is there a reason we shouldn't? (this is a genuine question, maybe I'm overlooking something...)

Reply:
@ValenberghsSven is right that the workflow where OVRB users create a case on which Secretarie adds subcases afterwards is not supported. It was indeed mentioned once in a meeting, but the conclusion back then was that those cases are so rare that we won't put effort in supporting it in the app. Therefore, I wouldn't bother about the possible data inconsistency this might introduce at the moment. 

Due to the unclear split between case/besluitvormingsaangelegenheid/publicatieaangelegenheid in the model at the moment, a lot of these kind of data inconsistencies are currently possible. I think it's more important to spend time in the near future to clean up the model and make it OSLO compliant than trying to to try to tie up all loose ends in the current model.

Reply:
Alright, then I concede my objection ;-)